build-files/artic/1.6.2/README.md

```diff
@@ -0,0 +1,319 @@
+  tensorboard              2.8.0         pyhd8ed1ab_1            conda-forge
+  tensorboard-data-server  0.6.1         py39hd97740a_4          conda-forge
+  tensorboard-plugin-wit   1.8.1         pyhd8ed1ab_0            conda-forge
+  tensorflow               2.8.0         cpu_py39h4655687_0      conda-forge
+  tensorflow-base          2.8.0         cpu_py39h7e02d9e_0      conda-forge
+  tensorflow-estimator     2.8.0         cpu_py39heaff504_0      conda-forge
+  termcolor                2.5.0         pyhd8ed1ab_1            conda-forge
+  tk                       8.6.13        noxft_h4845f30_101      conda-forge
+  tomli                    2.2.1         pyhd8ed1ab_1            conda-forge
+  tqdm                     4.67.1        pyhd8ed1ab_1            conda-forge
+  typing-extensions        4.12.2        hd8ed1ab_1              conda-forge
+  typing_extensions        4.12.2        pyha770c72_1            conda-forge
+  tzdata                   2025a         h78e105d_0              conda-forge
+  unicodedata2             16.0.0        py39h8cd3c5a_0          conda-forge
+  urllib3                  2.3.0         pyhd8ed1ab_0            conda-forge
+  werkzeug                 3.1.3         pyhd8ed1ab_1            conda-forge
+  whatshap                 1.7           py39h1f90b4d_1          bioconda   
+  wheel                    0.45.1        pyhd8ed1ab_1            conda-forge
+  wrapt                    1.17.2        py39h8cd3c5a_0          conda-forge
+  xopen                    2.0.2         pyh707e725_2            conda-forge
+  xz                       5.2.6         h166bdaf_0              conda-forge
+  yaml                     0.2.5         h7f98852_2              conda-forge
+  yarl                     1.18.3        py39h9399b63_1          conda-forge
+  zipp                     3.21.0        pyhd8ed1ab_1            conda-forge
+  zlib                     1.2.13        h4ab18f5_6              conda-forge
+  zlib-ng                  2.0.7         h0b41bf4_0              conda-forge
+  zstandard                0.23.0        py39h8cd3c5a_1          conda-forge
+  zstd                     1.5.6         ha6fb4c9_0              conda-forge
+```
+
+</details>
+
+
+There is also a very useful SOP: [https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html](https://artic.network/ncov-2019/ncov2019-bioinformatics-sop.html)
+And additional documentation: [https://artic.readthedocs.io/en/latest/](https://artic.readthedocs.io/en/latest/)
+
+> A bioinformatics pipeline for working with virus sequencing data sequenced with nanopore.
+
+WARNING : This container does not contain the primer schemes found at [https://github.com/artic-network/primer-schemes](https://github.com/artic-network/primer-schemes). Those will have to be downloaded and mounted separately. There are now commands to download the use the primers at runtime.
+
+This image DOES contain the medaka models downloaded with `artic_get_models`
```

Comment:
is it accurate to call these model files "medaka" models?

I thought medaka was deprecated in favor of Clair3, right?

If so, wouldn't we refer to them as "Clair3" models?

more info on Sam's blog post here: https://virological.org/t/fieldbioinformatics-updates/988

There's mentions throughout the Dockerfile and this readme that I think should be updated to say "Clair3 models" instead

Reply:
The models are Clair3 models (they have URLs like https://cdn.oxfordnanoportal.com/software/analysis/models/clair3/r1041_e82_260bps_fast_g632.tar.gz ) , so I've adjusted the documentation. Sorry!